silnlp/common/paratext.py

```diff
@@ -172,28 +173,62 @@ def extract_terms_list(list_type: str, project: Optional[str] = None) -> Dict[st
             if domain == "":
                 domain = "?"
             gloss_str = term_elem.findtext("Gloss", "")
-            match = re.match(r"\[(.+?)\]", gloss_str)
-            if match is not None:
-                gloss_str = match.group(1)
-
             refs_elem = term_elem.find("References")
             refs_list: List[VerseRef] = []
             if refs_elem is not None:
                 for verse_elem in refs_elem.findall("Verse"):
                     bbbcccvvv = int(verse_elem.text[:9])
                     refs_list.append(VerseRef.from_bbbcccvvv(bbbcccvvv))
                 references[id] = refs_list
-            gloss_str = gloss_str.replace("?", "")
-            gloss_str = clean_term(gloss_str)
-            gloss_str = re.sub(r"\s+\d+(\.\d+)*$", "", gloss_str)
-            glosses: List[str] = re.split("[;,/]", gloss_str)
-            glosses = [gloss.strip() for gloss in glosses if gloss.strip() != ""]
+            glosses = _process_gloss_string(gloss_str)
             terms_metadata_file.write(f"{id}\t{cat}\t{domain}\n")
             terms_glosses_file.write("\t".join(glosses) + "\n")
             terms_vrefs_file.write("\t".join(str(vref) for vref in refs_list) + "\n")
     return references
 
 
+def extract_major_terms_per_langauge(iso: str) -> None:
```

Comment:
You misspelled language in this function name.

Reply:
Fixed.